test/yamlContext/fixtures/basic.yaml

```diff
@@ -0,0 +1,76 @@
+
+# global site info
+site:
+  title: 'FastBannerSigns.com'
+  # users can put arbitrary data here
+  fb-info: 'Info to show to facebook crawler'
+  twitter-info: 'Info that twitter crawler grabs'
+
+
+# pages names match hbs files
+pages:
+  home:
+    needs:
+      # use adapters to fetch data from services
+      # adapters are called by name (take2 for example) and they receive the attributes
+      # declared here. They return a promise that resolves in the data needed.
+      # This resolved data is merged into the context to be used by the corresponding
+      # hbs template file
+      take2:
+        type: 'group'
+        slug: 'large-banners'
+        as: 'featured-banners'
+
+      take2:
+        type: 'group'
+        slug: 'small-signs'
+        as: 'featured-signs'
+
+      stripe:
+        call: 'payments.list'
+        as: 'payments'
+
+      url:
+        path: 'http://api.github.com/pulls?user=take2'
+        as: 'pull-requests’
+
+  freeway-signs:
+    needs:      
+      # default behavior of categories
```

Comment:
oh no i understand now.  you're just illustrating the default.  that could be removed with the same effect